### ISSUE TITLE
Option to Enable logging and How long to keep these logs

### ISSUE THREAD

**Issue by ngcoders** [baseapp/wpoven_suresend#2]
Currently all Emails are logged , There should be an option to enable / disable them and also how long to keep them .

**Comment by shyambaseapp** [comment]
You’re absolutely right — logging all emails by default can impact database size over time. To address this, we’ve added:

An option to enable/disable email logging

Configurable log retention periods (7 days, 30 days, 90 days, or unlimited)

Improved cleanup performance to safely remove old logs

This gives site owners full control over whether emails are logged at all, and for how long they are retained, based on their needs and server capacity.
